### DISCUSSION TITLE
alt-tab / cmd-tab

### DISCUSSION THREAD

**Discussion by kulfix** [nikitabobko/AeroSpace#722] [questions-and-answers]
Is it possible to make cmd-tab work like showing only apps from the current workspace? It can be another shortcut.

**Comment by kamaradclimber** [comment]
That would be great to have a command to switch to "the next" window in the current workspace.
(especially when using accordeon layout)

**Comment by philsnow** [comment]
I doubt you'll be able to make Macos's cmd-tab only show apps that are on the current Aerospace workspace, but if you're open to using something like Hammerspoon, you can do something like this: https://gist.github.com/philsnow/29a5c557bd6c7f513c3aef9dc41280be

(note, it draws a window switcher, not an app switcher, so it's less like Macos's native cmd-tab switcher)
